### PR TITLE
MS. Canon. Misc. 573: New description

### PR DESCRIPTION
**Pull request by adunning**
Adapted from [an edition using this manuscript](https://www.jstor.org/stable/1006612) – still missing some standard information, but will fill these in when there is an opportunity.

### CODE REVIEW

File: collections/Canon_Misc/MS_Canon_Misc_573.xml [hunk 1]
```diff
@@ -71,82 +80,191 @@
...
+                           </layout>
+                        </layoutDesc>
                      </objectDesc>
                   </physDesc>
                   <history>
                      <origin>
-                        <origDate calendar="Gregorian" when="1385">1385</origDate>
-                        
-                     </origin>
+                        <origPlace key="place_7006464">Prague</origPlace>
```
Comment (holfordm) [inline collections/Canon_Misc/MS_Canon_Misc_573.xml:167]: this should be in the format:
'<origPlace><country key="xxxxxx">Bohemia</country>, <settlement key="place_7006464">Prague</settlement><origPlace>'

File: collections/Canon_Misc/MS_Canon_Misc_573.xml [hunk 2]
```diff
@@ -71,82 +80,191 @@
...
+                                 <height>225</height>
+                                 <width>153</width>
+                              </dimensions>
+                           </layout>
+                        </layoutDesc>
                      </objectDesc>
                   </physDesc>
                   <history>
                      <origin>
-                        <origDate calendar="Gregorian" when="1385">1385</origDate>
```
Comment (holfordm) [inline collections/Canon_Misc/MS_Canon_Misc_573.xml:101]: add `@cert="high"` for dated mss.

File: collections/Canon_Misc/MS_Canon_Misc_573.xml [hunk 3]
```diff
@@ -71,82 +80,191 @@
...
-                        
-                     </origin>
+                        <origPlace key="place_7006464">Prague</origPlace>
+                        <origDate calendar="Gregorian" when="1385">1384–85</origDate>, as indicated through closing rubrics for each book.</origin>
```
Comment (holfordm) [inline collections/Canon_Misc/MS_Canon_Misc_573.xml:168]: add `@cert="high"` for dated mss

**Comment by holfordm** [comment]
just a couple of minor comments. Nice catch linking this with MS. Canon. Pat. Lat. 52!

**Comment by adunning** [comment]
Thanks for catching these – should be ready now.

**Pull request merged**